### PR TITLE
Fixes #11 - Add new strategies

### LINKED ISSUE #11: New strategies
**Is your feature request related to a problem? Please describe.**
Players need strategies more intelligent than _basic_. 

**Describe the solution you'd like**
Implement a greedy player that make a choice based on the points that will could score.
Implement a player that love to full vertical lines of their table.
...

**Additional context**
After finish a new strategy, don't forget to add the strategy to the _random_strategy_ method list

### PR DESCRIPTION
**Pull request by AntiD2ta**
Fix #11 

**Changes**:
- Add a greedy strategy
- Add a fill column wise strategy
- Fix some minor bugs related to test, logs and documentation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.PHONY : doc, run, test, coverage, help
```
Comment (e1Ru1o) [inline Makefile]: Only doc needs to be PHONY
```suggestion
.PHONY : doc
```

File: Makefile [hunk 2]
```diff
@@ -8,7 +10,7 @@ run: ## Run the game
 	@swipl --quiet -t "main($(level), \"$(file)\", $(players), $(fac))" -s game.pl 
     
 test: ## Run tests
-	@swipl --quiet -t "load_test_files([]), run_tests." -s game.pl 
+	@swipl -t "load_test_files([]), run_tests." -s game.pl
```
Comment (e1Ru1o) [inline Makefile]: Change to top level by goals in order to hide the swipl welcome message
```suggestion
	@swipl -g "load_test_files([]), run_tests." -g halt -s game.pl 
```

File: Makefile [hunk 3]
```diff
@@ -8,7 +10,7 @@ run: ## Run the game
 	@swipl --quiet -t "main($(level), \"$(file)\", $(players), $(fac))" -s game.pl 
     
 test: ## Run tests
-	@swipl --quiet -t "load_test_files([]), run_tests." -s game.pl 
+	@swipl -t "load_test_files([]), run_tests." -s game.pl 
  
 coverage: ## Run tests and display coverage
 	@swipl --quiet -t "use_module(library(test_cover)), load_test_files([]), show_coverage(run_tests)." -s utils.pl -s game.pl
```
Comment (e1Ru1o) [inline Makefile]: Change to top level by goals in order to hide the swipl welcome message. And remove duplicated -s
```suggestion
	@swipl -g "use_module(library(test_cover)), load_test_files([]), show_coverage(run_tests)." -g halt -s game.pl
```

File: player.pl [hunk 4]
```diff
@@ -88,16 +88,34 @@
...
+        property_of(valid, Line, ValidColors),
+        property_of(Fid, Fac, CurFac),
+        member(Color, ValidColors),
+        member(Color, CurFac)
+    ), Choices),
+    not(length(Choices, 0)).    
+
+%% available_colors(+Game:Game, -Choices:list) is det
+% 
+% The valid_choices/2 predicate given a Game find all
```
Comment (e1Ru1o) [inline player.pl]: Docstring error
```suggestion
% The available_colors/2 predicate given a Game find all
```

File: Makefile (hunk 2 above)
Comment (AntiD2ta) [reply Makefile]: Done in last commit

File: player.pl (hunk 4 above)
Comment (AntiD2ta) [reply player.pl]: Done in last commit

**Review by e1Ru1o — APPROVED** [review]

**Pull request merged**